CloudSdkCapacitorPlugin.podspec

```diff
@@ -11,8 +11,7 @@ Pod::Spec.new do |s|
   s.author = package['author']
   s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
-  s.ios.deployment_target  = '11.0'
+  s.ios.deployment_target  = '12.0'
   s.dependency 'Capacitor'
-  s.dependency 'PACECloudSDK'
```

Comment:
Is this not needed anymore? Where is the dependency listed then when developing for iOS?

Reply:
added this